roles/openshift_service_catalog/tasks/generate_certs.yml

```diff
@@ -35,6 +45,15 @@
     signer_key: "{{ generated_certs_dir }}/ca.key"
     signer_serial: "{{ generated_certs_dir }}/apiserver.serial.txt"
 
+- name: Generating Controller Manager keys
+  oc_adm_ca_server_cert:
```

Comment:
Why not use service serving cert for these?

Reply:
@smarterclayton are you talking about using the service annotation  service.alpha.openshift.io/serving-cert-secret-name  to create the key and cert for us?  That is how we did it upstream in service catalog, but here with Ansible we were already doing it this way for our api server so I just followed the pattern.